layers/core_checks/cc_image.cpp

```diff
@@ -218,9 +218,17 @@ bool CoreChecks::PreCallValidateCreateImage(VkDevice device, const VkImageCreate
         if (!LvlFindInChain<VkExternalFormatANDROID>(pCreateInfo->pNext)) {
 #endif  // VK_USE_PLATFORM_ANDROID_KHR
             skip |= LogError(device, "VUID-VkImageCreateInfo-imageCreateMaxMipLevels-02251",
-                             "vkCreateImage(): Format %s is not supported for this combination of parameters and "
-                             "VkGetPhysicalDeviceImageFormatProperties returned back %s.",
-                             string_VkFormat(pCreateInfo->format), string_VkResult(result));
+                             "vkCreateImage(): The following parameters -\n"
+                             "format (%s)\n"
+                             "type (%s)\n"
+                             "tiling (%s)\n"
+                             "usage (%s)\n"
+                             "flags (%s)\n"
+                             "returned a non-success result (%s) when calling VkGetPhysicalDeviceImageFormatProperties (which "
+                             "makes all things undefined).",
```

Comment:
IMO this reads better. Since if you are getting a validation error it's kinda implied the result isn't successful. Also the `which makes all things undefined` isn't really needed IMO. Since the VUID already says that. Making it kinda redundant.

```suggestion
                             "returned (%s) when calling VkGetPhysicalDeviceImageFormatProperties.",
```

Reply:
agree, I guess this VU is just aweful to understand, but happy with your suggestions

Reply:
Yes I also think this VU is awful to read/understand.